pkg/preflights/host-preflight.yaml

```diff
@@ -886,3 +871,12 @@ spec:
           - pass:
               when: 'net.ipv4.conf.all.arp_ignore == 0'
               message: "ARP ignore is disabled for all interfaces on the host."
+    - sysctl:
+        checkName: "IP forwarding"
+        outcomes:
+          - fail:
+              when: 'net.ipv4.ip_forward == 0'
+              message: "IP forwarding must be enabled. To enable it, edit /etc/sysctl.conf, add or uncomment the line 'net.ipv4.ip_forward=1', and run 'sudo sysctl -p'."
+          - pass:
+              when: 'net.ipv4.ip_forward > 0'
```

Comment:
According to the docs - https://www.kernel.org/doc/html/latest/networking/ip-sysctl.html:
> ip_forward - BOOLEAN
>
>        0 - disabled (default)
>
>        not 0 - enabled